src/tests/ftest/util/osa_utils.py

```diff
@@ -310,6 +310,31 @@ def set_cont_class_properties(self, oclass="S1"):
             prop = prop.replace("rf:1", rf_value)
         self.container.properties.value = prop
 
+    def assert_on_exception(self, out_queue):
```

Comment:
It looks like you're calling this method with self.out_queue as parameter, so can't we just make this method no argument and use self.out_queue?

Reply:
Agree... Sure. I can do the change....

Reply:
Okay, so in that case, shouldn't we use out_queue instead of self.out_queue on line 320?

Reply:
Yes. Good find... will fix it...